docs/deploy/setup-iomesh.md

```diff
@@ -84,12 +84,15 @@ spec:
 ```
 
 #### Mount Type
-
-IOMesh provides 2 mount types now:
+In `mixed-flash` deployment node, IOMesh provides 2 mount types:
 
 - `cacheWithJournal`: used for performance tier of storage pool. It **MUST** ba a partitionable block device. IOMesh will partition the device into 2 paritions: one for `journal` and another for `cache`. Either `SATA` or `NVMe` SSD is recommanded.
 - `dataStore`:  used for capacity tier of storage pool. Either `SATA` or `SAS` HDD is recommanded.
 
+In `all-flash` deployment node, IOMesh only provides 1 mount type:
+
+- `dataStoreWithJournal`: used for capacity tier of storage pool. It **MUST** ba a partitionable block device. IOMesh will partition the device into 2 paritions: one for `journal` and another for `cache`. Either `SATA` or `NVMe` SSD is recommanded.
+
```

Comment:
**MUST** ba --> **MUST** be
2 paritions --> 2 partitions
one for `journal` and another for `cache` --> one for `journal` and the other for `dataStore`